src/containers/Error.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Center, HorizontallyCenter } from "../components/styled";
+import { Text } from "@blueprintjs/core";
+
+export const ErrorContainer = () => {
+  return (
+    <Center>
+      <HorizontallyCenter>
+        <Text>
+          What?! please restart the Wallet{" "}
```

Comment:
Right now, this will be show if the user tries to edit the settings and the GrinNode is not running. This container is meant to be displayed when an unmanaged error occur. I will add the proper error.